src/consensus/tower.zig

```diff
@@ -40,12 +40,20 @@ const ComputedBankState = struct {
 
 pub const ThresholdDecision = union(enum) {
     passed_threshold,
-    failed_threshold: struct {
-        // vote depth
-        u64,
-        // Observed stake
-        u64,
-    },
+    failed_threshold: FailedThreshold,
+
+    /// NOTE: this is a tuple in the original rust code
+    pub const FailedThreshold = struct {
+        vote_depth: u64,
+        observed_stake: u64,
+    };
+
+    /// #[default]
+    pub const DEFAULT: ThresholdDecision = .passed_threshold;
```

Comment:
Why is this the default? Can't you just use `.passed_threshold` directly when you want the "default"?

Reply:
> Why is this the default?

Same applies https://github.com/Syndica/sig/pull/851/files#r2275331731

I can allude to the why, but @InKryption will be best to ask for choosing this style. This is only showing here as a new diff as it was moved to replace the duplicated version.

Reply:
[792d00d](https://github.com/Syndica/sig/pull/851/commits/792d00dc3c8837344fc7fe7550983d66ba15273b)

Reply:
Originally just ported over from agave, which specified `passed_threshold` as the default value. I have no attachment to it, it was simply convenient while reasoning about code that I was in the process of porting over.